### DISCUSSION TITLE
Add controls to Web Log (or add a custom web page with controls)

### DISCUSSION THREAD

**Discussion by g-mocken** [HomeSpan/HomeSpan#747] [General]
Would it be possible to add simple controls to the web log page, for example in the way the esphome project does?

Or is it possible to simultaneously host a simple custom web page for such a purpose?
I would like to control my homespan-driven HomeKit devices also from a non-Apple device.

**Comment by HomeSpan** [comment]
Adding your own web server is the easiest  and most flexible way to accomplish this.  See this [FAQ](https://github.com/HomeSpan/HomeSpan/blob/master/docs/FAQ.md#can-you-add-a-web-server-to-homespan) and references therein.
